apps/music/views/albums/view.js

```diff
@@ -77,6 +77,7 @@ AlbumsView.prototype.getAlbums = function() {
         .then(response => response.json())
         .then((albums) => {
           albums.forEach((album) => {
```

Comment:
Let's do a `map()` call here instead to produce a new array and just copy the properties we need to new objects. Jim suggested we do this for search and it seems to work ok there. His reasoning was that the original array and heavy metadata objects would get GC'd and it may end up using less memory.